metabase/metabase
# Use Box to style components in the app

Use the recommended approach to style app components (e.g. not ones in the `ui` folder). Key points:
- styles are defined via `createStyles` call as recommended in the docs https://mantine.dev/styles/create-styles/ `We recommend using createStyles to create styles for the rest of your application as it provides the most convenient way to utilize Mantine theme`
- styles are placed in a separate `*.styles.ts` file as in the `mantine` source code https://github.com/mantinedev/mantine/blob/master/src/mantine-core/src/Badge/Badge.tsx and https://github.com/mantinedev/mantine/blob/master/src/mantine-core/src/Badge/Badge.styles.ts. The only difference is that they use default exports for styles and we don't.
- removes `emotion` import source as it was intended for emotion's `css` prop, but now we're using the mantine apis.

How to verify:
- open the login page, make sure it looks like in `master`

<img width="571" alt="Screenshot 2023-06-08 at 13 04 57" src="https://github.com/metabase/metabase/assets/8542534/d69fd5ad-987d-4ed1-b196-f3f216ffa639">
